### ISSUE TITLE
Create a pdf of a4 page with number of QR codes on each page.

### ISSUE THREAD

**Issue by DancingQuanta** [balos1/easy-gpg-to-paper#4]
What is the minimum size of QR code produced? Can two QR codes sit next to each other across short lenght of a4 page and be distinguished?

The basic idea to put all QR codes into a single pdf and that pdf can be piped into `lp` print.

**Comment by balos1** [comment]
There should not be a problem putting multiple QR codes next to each other on an a4 page. The way I am doing it currently, the version of the QR code (which determines the amount of data it can store), is determined automatically based on the data requirement... so I am not sure what is the exact minimum size is achievable. 

I am considering adding the ability to output the codes as a single pdf, as I also think it would be useful. The idea behind having the produced qrcodes separated was because a lot of people (who are extra paranoid) like to keep the different segments in different physical locations.

**Comment by DancingQuanta** [comment]
Ah I did not think of keeping QR codes separate! But you can cut the paper up with scissors, no need to waste a whole sheet for one QR code.
I believe there should be an option where a number of QR codes is printed per A4 page paper. 

One could random rearrange the QR code on a paper.... No idea how to put that back together!

**Comment by balos1** [comment]
Yeah, that is a great point. I will try to add the feature in the next version, which I am working on over the next couple weeks.

**Comment by nurupo** [comment]
@DancingQuanta https://github.com/nurupo/paper-store allows you to print 4 QR codes on a single page. You could probably fit 6, but you'd need to downscale QR code images a bit and the page would be way too crammed to fit header, footer and QR code numbered labels indicating the order in which to put them together.
